### PR TITLE
Remove CentOS from the platforms list

### PR DESCRIPTION
**Pull request by jfjoly**
Signed-off-by: JF Joly <jfjoly@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ analysis that are not present in vanilla PostgreSQL. (For example, the `time_buc
 TimescaleDB is available pre-packaged for several platforms:
 
 - Linux:
-    - [RedHat / CentOS](https://tsdb.co/GitHubTimescaleRedHatCentOS)
+    - [RedHat](https://tsdb.co/GitHubTimescaleRedHatCentOS)
```
Comment (jnidzwetzki) [inline README.md:122]: Nit: Maybe we should change the name of the redirect as well (`GitHubTimescaleRedHatCentOS`)

**Review by jnidzwetzki — APPROVED** [review]

**Review by svenklemm — CHANGES_REQUESTED** [review]
Why are we changing this? We are still building packages for CentOS

**Comment by Tenorotti** [comment]
nudging this to get it reviewed and approved

**Comment by svenklemm** [comment]
> nudging this to get it reviewed and approved

We are still using Centos 7 to build our RedHat packages for that version so the change is incorrect IMO.

**Pull request closed without merging**